event_operating_unit/views/event_registration_views.xml

```diff
@@ -18,15 +18,10 @@
         <field name="arch" type="xml">
             <search>
                 <field name="operating_unit_id" string="Operating Unit" groups="operating_unit.group_multi_operating_unit" />
```

Comment:
You should use an xpath expression or at least a field with `position` attribute to add this field.